code/modules/uplink/uplink_items.dm

```diff
@@ -2040,6 +2058,14 @@ GLOBAL_LIST_INIT(illegal_tech_blacklist, typecacheof(list(
 	surplus = 0
 	purchasable_from = UPLINK_NUKE_OPS
 
+/datum/uplink_item/implants/cultist
+	name = "Ancient Blood"
+	desc = "Contains the Ancient Blood of a Blood God."
+	item = /obj/item/implanter/cultist
```

Comment:
probably a typo as it is undefined type path?

Reply:
No, forgot to add it to the dme.